hudi-flink/src/main/java/org/apache/hudi/table/catalog/HoodieCatalog.java

```diff
@@ -103,10 +103,10 @@ public void open() throws CatalogException {
     catalogPath = new Path(catalogPathStr);
     try {
       if (!fs.exists(catalogPath)) {
-        throw new CatalogException(String.format("Catalog %s path %s is not exists.", getName(), catalogPathStr));
+        throw new CatalogException(String.format("Catalog %s path %s does not exists.", getName(), catalogPathStr));
       }
     } catch (IOException e) {
-      throw new CatalogException(String.format("Check catalog path %s exists occur error.", catalogPathStr), e);
+      throw new CatalogException(String.format("Check catalog path %s exists occur exception.", catalogPathStr), e);
     }
```

Comment:
If you want to unify the error message format, i would propose this style:
xxxing xxx error(or exception), for e.g, we can refactor this error message to:
`Checking whether catalog path %s exists exception."`